content/04.study.md

```diff
@@ -47,6 +47,25 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Introduction 
+
+DNA Methylation (DNAm), is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 
+This DNA-level epigenetic modification regulates gene transcription, is critical in development, and alterations to DNA methylation are well-established as contributing to pathophysiology of many diseases including cancers. 
```

Comment:
☝️ needs a reference - it's common knowledge in some places, but it'd be good to point folks to one or more topical papers (maybe even a few + a nice summary review if you have one) since some of our readers will be ML folks, not biologists.

My impression is also that DNA methylation can also be affected by processes that alter genome accessibility, including transcription, and so it can also provides a readout of a cell's past states. If this isn't totally off base, it's probably good to mention it here too. Doesn't the TP53 status thing fall into this category?

Reply:
added references here for the basics of DNA methylation biology and methylation in disease. The point about genome accessibility is true in that there is a very high correlation of DNA methylation and histone/chromatin measures for accessibility/inaccessibility, though the relationship with time and cell states has mostly been studied in the context of lineage commitment during development and there is not nearly enough data within specific cell types to trace past cell states yet.